### ISSUE TITLE
'Select' button doesn't work for the first position on the song list

### ISSUE THREAD

**Issue by RenBabiuch** [Asvarox/allkaraoke#318]
**Description:**
After selecting a song on the remote mic, a song's preview is visible in the desktop app - except for the first song on the song list (no reaction).

**Steps to reproduce:**
>1. Enter the game
>2. As input type - choose 'Use smartphones' and connect device
>3. On the desktop app: 
>a. go to Main Menu
>b. click the "Sing song" button
>c. ensure all  song languages are selected
>d. go back to Song List
>4. On the remote mic app: 
>a. go to the Song List
>b. click on the 'flag' to select song language
>c. ensure all  song languages are selected
>d. go back to Song List
>5. Select the first song and click the 'SELECT' button next to the song title

**Expected behaviour:**
After selecting a song on the remote mic, a preview of that song should be visible in the desktop app.

**Current behaviour:**
Song previews are visible, except for the first song on the list. When I  click the 'SELECT' button, nothing happens.

**Comment by Asvarox** [comment]
@RenBabiuch should be fixed on prod, please reverify and close if it doesn't reproduce

**Comment by RenBabiuch** [comment]
@Asvarox  Yes, the preview is visible for all songs - including the first song. I close this issue. Thanks :)
